README.md

```diff
@@ -206,11 +230,11 @@ you may write the following mapping:
 ```yaml
 row:
    map:
+      # When no column is indicated, OntoWeaver will map the indice of the row to the node name.
```

Comment:
Map actually raises a `ValueError(f"No column declared for the {type(self).__name__} transformer, did you forgot to add a columns keyword?")` when no columns are indicated in the input mapping, since we added the `rowIndex` transformer class to handle the case when one wants to use the index of the input row as ID for the node.